components/shared/ConnectWalletButton.tsx

```diff
@@ -137,10 +140,11 @@ const ConnectWalletButton: FC<ConnectWalletButtonProps> = ({
 					"&:hover": {
 						color: "white",
 						backgroundColor: color,
+						opacity: "1",
 					},
 				}}
 				onClick={() => {
-					onClick;
+					onClick();
 				}}
```

Comment:
this can be collapse to `onClick={onClick}`

Reply:
should be fixed now